content/en/tracing/other_telemetry/connect_logs_and_traces/php.md

```diff
@@ -82,7 +82,7 @@ use Illuminate\Support\Facades\Log;
 Log::info('Hello, World! [%dd.trace_id% %dd.span_id% %status%]');
 ```
 
-The PHP tracer replaces the placeholders with the corresponding values. For example, the logged message above could be transformed into:
+The PHP tracer replaces the placeholders with the corresponding values. For example, the logged message above will be transformed into:
```

Comment:
```suggestion
The PHP tracer replaces the placeholders with the corresponding values. For example, the logged message above could be transformed into:
```

Reply:
Thanks for clarifying @cecile75, I've updated the recommendation. My point here is to avoid use of future tense, which is not recommended per the style guide.

Should this also be updated on line `58`, which likewise uses `could be`?

Reply:
line 58 is little different in my opinion.
With option 1, any trace correlation identifier with a value at the time of the log will be injected. So it's really more a "could" as the user doesn't chose explicitly which trace correlation identifiers will be injected.
While with option 2, as the coder explicitly choose to inject %dd.trace_id% %dd.span_id% %status% , I think we can assume that these items have a value at this time of the log and will be injected and present in the resulting log.